Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/function/editor/ParamInfo.java

```diff
@@ -160,9 +160,6 @@ Parameter getParameter(boolean isCustom) {
 			}
 			// preserve original source type if name unchanged
 			SourceType source = SourceType.USER_DEFINED;
-			if (original != null && original.getName().equals(name)) {
-				source = original.getSource();
-			}
 			return new MyParameter(name, formalDataType, variableStorage, model.getProgram(),
 				source);
```

Comment:
source can just be replaced with SourceType.USER_DEFINED here.

Reply:
Yeah, this can additionally be done, correct